code/modules/power/smes.dm

```diff
@@ -160,6 +156,13 @@
 
 	return ..()
 
+/obj/machinery/power/smes/wirecutter_act(mob/living/user, obj/item/I)
+	//disassembling the terminal
+	if(terminal)
```

Comment:
Also you removed the requirement of panel_open

Reply:
fixed